docs/pages/reference/session-recording.mdx

```diff
@@ -0,0 +1,179 @@
+---
+title: Teleport Session Recording
+description: An overview of Teleport's session recording and its configuration
+---
+
+This page provides an overview of how Teleport records sessions and the various
+configuration options that apply to session recordings. The content that is
+captured depends on the type of the session.
+
+- SSH sessions: Teleport captures the entire pseudo-terminal (PTY) output
+  of the session
+- Kubernetes sessions: Teleport captures the entire PTY output for
+  `kubectl exec` invocations.
+- Desktop sessions: Teleport captures the contents of the desktop screen
+  and any mouse input. Teleport does not capture keystrokes in the remote
+  desktop.
+- App sessions: Teleport captures a stream of `app.session.request`
+  audit events related to application access. The audit events are
+  bucketed into 5-minute time intervals informally referred to as "chunks."
+- Database sessions: Teleport captures a stream of audit events
+  related to the database being accessed.
+
+Teleport supports two different recording configurations: synchronous and
+asynchronous recording. Additionally, Teleport administrators can configure
+where the recording takes place: at the node or at the proxy.
+
+This results in 4 possible session recording configurations:
+
+- `node-sync`: synchronous recording performed by an SSH Service instance
+- `node`: asynchronous recording performed by an SSH Service instance
+- `proxy-sync`: synchronous recording performed on a Proxy Service instance
+- `proxy`: asynchronous recording performed on a Proxy Service instance
+
+This is a cluster-wide configuration option and applies to the entire
+Teleport cluster. It can be configured in by setting `session_recording`
+in the `auth_service` section of your `teleport.yaml`, or dynamically via
+the the `session_recording_config` resource. If you need to apply different
+recording configuration to different sets of resources, we recommend setting up
+[Trusted Clusters](/docs/management/admin/trustedclusters/) with their own
+recording configurations.
+
+<Admonition type="note">
+Windows, database, and Kubernetes sessions are always recorded at the host running
+the Teleport service for this session type, since there is no Teleport software
+running on the "node" (desktop, database, or Kubernetes cluster).
+
+For this reason, Teleport Windows, Database, and Kubernetes Access treats `node`
+and `proxy` identically (perform synchronous recording) and `node-sync` and
+`proxy-sync` identically (perform asynchronous recording).
+</Admonition>
+
+## Record at Node
```

Comment:
```suggestion
### Record at Node
```
To place this in the "Session recording configurations" section I suggested earlier